tidb-server/main.go

```diff
@@ -178,15 +178,19 @@ func main() {
 	signal.SetupSignalHandler(serverShutdown)
 	runServer()
 	cleanup()
-	exit()
+	syncLog()
 }
 
 func exit() {
+	syncLog()
```

Comment:
This change seems unrelated

Reply:
`exit()` is also called by `setCPUAffinity()`. This is to make sure the logic is the same for `setCPUAffinity()`

https://github.com/pingcap/tidb/blob/0ff5b50c1509bed279a2c2ba7acdb38519b38c9c/tidb-server/main.go#L192